ThunderstoreCLI.Tests/Utils/Comparers.cs

```diff
@@ -4,58 +4,58 @@ namespace ThunderstoreCLI.Tests
 {
     public class ThunderstoreCLI_Comparers_SemVer
     {
-        private readonly ThunderstoreCLI.Comparers.SemVer _semVer = new ThunderstoreCLI.Comparers.SemVer();
+        private readonly Comparers.SemVer _semVer = new();
 
-        public static TheoryData<int[], int[]> EqualValues => new TheoryData<int[], int[]>
+        public static TheoryData<int[], int[]> EqualValues => new()
         {
-            { new [] { 1, 0, 0 },    new [] { 1, 0, 0 } },
-            { new [] { 0, 1, 0 },    new [] { 0, 1, 0 } },
-            { new [] { 0, 0, 1 },    new [] { 0, 0, 1 } },
-            { new [] { 0, 0, 0 },    new [] { 0, 0, 0 } },
-            { null,                  null }
+            { new[] { 1, 0, 0 }, new[] { 1, 0, 0 } },
+            { new[] { 0, 1, 0 }, new[] { 0, 1, 0 } },
+            { new[] { 0, 0, 1 }, new[] { 0, 0, 1 } },
+            { new[] { 0, 0, 0 }, new[] { 0, 0, 0 } },
+            { null, null }
         };
 
         [Theory]
         [MemberData(nameof(EqualValues))]
         public void ReturnZero_WhenValuesAreEqual(int[] a, int[] b)
         {
-            var result = _semVer.Compare(a, b);
+            int result = _semVer.Compare(a, b);
```

Comment:
Most of this PR seems to be removing all uses of `var`, is this a code style change that we want? If so, should we enforce this in the future with an editorconfig?

Reply:
With 50d00bf7a052adc8da0b54ccf211dd3f9b096873 a full .editorconfig has been added that enforces this rule.

Personally I like explicit types unless the type is clear from the right side, which the current settings enforce, but this can be changed as desired.